tests/functional/modules/test_zos_find_func.py

```diff
@@ -617,3 +617,23 @@ def test_find_sequential_special_data_sets_containing_single_string(ansible_zos_
     finally:
         for ds in special_names:
             hosts.all.shell(cmd=f"drm '{ds}'")
+
+
+def test_find_migrated_data_sets(ansible_zos_module):
+    hosts = ansible_zos_module
+    find_res = hosts.all.zos_find(
+        patterns=['IMSBLD.I15STSMM.*','IMSBLD.DCC71QPP.*'],
+        resource_type='migrated'
+    )
+    for val in find_res.contacted.values():
+        assert len(val.get('data_sets')) != 0
+
+def test_find_migrated_data_sets_with_excludes(ansible_zos_module):
+    hosts = ansible_zos_module
+    find_res = hosts.all.zos_find(
+        patterns=['IMSBLD.I15STSMM.*','IMSBLD.DCC71QPP.*'],
+        resource_type='migrated',
+        excludes='.*F4'
+    )
+    for val in find_res.contacted.values():
+        assert len(val.get('data_sets')) != 0
```

Comment:
Please add an assertion that makes sure there are no data sets that you are trying to exclude like :
`IMSBLD.DCC71QPP.JMK7702.F4`, this can be done in many ways, like using an iterator and an anonymous function to make sure there are no matches to that regex.